charts/incubator/alist/values.yaml

```diff
@@ -0,0 +1,18 @@
+image:
+  repository: tccr.io/truecharts/alist
+  pullPolicy: IfNotPresent
+  tag: v2.1.0@sha256:5aaf09a89ce257292d29260bd3e857d1ebf79d7bcf94db2ce8e64d7c0743a57a
```

Comment:
Update digest once image is made public in containers.

Reply:
ok